packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -145,6 +146,114 @@ const classesForContent = (rowIndex, rowCount, contentIndex, emphasis) => {
+  );
+};
+
+const renderTwoColumns = (displayGroup, displayGroupIndex, emphasis, overrideDefaultStyling) => {
+  const columnKey = displayGroupIndex;
+  const displayCount = displayGroup.length;
+  const containerStyling = displayGroupIndex === 0 ? 'primary-column' : 'secondary-column';
+
+  return (
+    <li className={cx(containerStyling)} key={columnKey}>
+      <ul className={cx('column-list')} key={columnKey}>
+        {displayGroup.map((display, contentIndex) => {
+          const contentKey = contentIndex;
+          const contentClasses = overrideDefaultStyling ? 'content' : classesForContent(contentIndex, displayCount, displayGroupIndex, emphasis);
+
+          return (
+            <li className={cx(contentClasses)} key={contentKey}>
+              {display}
+            </li>
+          );
+        })}
+      </ul>
+    </li>
+  );
+};
+
+const renderColumn = (displays, emphasis, overrideDefaultStyling, header) => {
+  const displayCount = displays.length;
+  const headerID = `single-column-header-${uuidv4()}`;
+
+  return (
+    <div>
+      <h6 id={headerID} hidden>{header}</h6>
+      <ul aria-labelledby={headerID} className={cx('column-list-container')}>
+        {displays.map((display, displayIndex) => {
+          const contentKey = displayIndex;
+          /**
+           * We are only ever rendering one column so zero is being passed into classesForContent for the contentIndex variable.
+           * classesForContent would usually take in an index for that but in this case we don't have one explicitly.
+           */
+          const contentClasses = overrideDefaultStyling ? 'content' : classesForContent(displayIndex, displayCount, 0, emphasis);
+
+          return (
+            <li className={cx(contentClasses)} key={contentKey}>
+              {display}
+            </li>
+          );
+        })}
+      </ul>
+    </div>
+  );
+};
+
+const renderByRowView = (displays, emphasis, overrideDefaultStyling, header) => {
+  const displayGroups = [];
+  const headerID = `by-row-header-${uuidv4()}`;
+
+  while (displays.length) {
+    displayGroups.push(displays.splice(0, 2));
+  }
+
+  return (
+    <div>
+      <h6 id={headerID} hidden>{header}</h6>
+      <ul aria-labelledby={headerID} className={cx('row-list-container')}>
+        {displayGroups.map((displayRow, rowIndex) => {
+          const row = renderRow(displayRow, rowIndex, displayGroups.length, emphasis, overrideDefaultStyling);
+          return row;
+        })}
+      </ul>
+    </div>
+  );
+};
+
+const renderSingleDisplayView = (singleDisplay, overrideDefaultStyling) => {
+  /**
+   * Since this is always a singular display, the content styling will be the primary defaults if they are not overridden.
+   * We don't have to call into the classesForContent method and instead can just set the primary size and color here.
+   */
+  const contentClass = overrideDefaultStyling ? 'content' : ['content'].concat(['content-primary-size', 'content-primary-color']);
```

Comment:
nit: do we need to use `concat` here or can we just use the following?
```
['content', 'content-primary-size', 'content-primary-color']
```

Reply:
Good suggestion, changed here: [073a1ab](https://github.com/cerner/terra-clinical/pull/913/commits/073a1abb106f1bceb6672e2cd92ad8ce27fa17b2)